src/test_typing.py

```diff
@@ -1070,6 +1070,10 @@ class Node(Generic[T]): ...
             self.assertEqual(t, copy(t))
             self.assertEqual(t, deepcopy(t))
             if sys.version_info >= (3, 3):
+                # From copy module documentation:
+                # It does “copy” functions and classes (shallow and deeply), by returning
```

Comment:
Thanks! Just one more thing: we generally prefer not to use non-ASCII characters in stdlib (and this repo is upstream for stdlib). Here you copied quotes from the HTML docs that are non-ASCII, could you please switch to ASCII quotes?